content/docs/integrations/kubernetes/index.md

```diff
@@ -0,0 +1,155 @@
+---
+title: Adding a Kubernetes cluster
+lastUpdated: '2021-07-15T21:00:00.0Z'
+description: How to add a Kubernetes cluster for the Kubernetes plugin.
+---
+
+[Full active cluster on backstage](./active.png)
+
+> Note, the Kubernetes plugin is only supported with AWS clusters.
+> We are currently working on trying to get GKE as another supported provider.
+
+# AWS
+
+## Introduction
+
+In order to use the Kubernetes plugin, Roadie needs:
+ * An Assumed Role to fetch the resources from your cluster
+ * The name of your cluster
+ * URL of the loadbalancer/entry point to your kubernetes cluster
+
+
+These are set within backstage at the following url:
+
+```text
+https://<tenant-name>.roadie.so/administration/settings/kubernetes
+```
+
+This page describes how to create and set up the API token.
+
+## Steps
+
+### Step 1: Creating the cross federation role
+
+Start off by signing into your AWS console and navigating to the IAM service.
+Once there, click on `Role` link (this should be on the left handside of your screen).
+
+Now that we are in the Role tab, please click on the `Create Role` button.
+
+Click on `Another AWS Account` and add the account number on on the kubernetes configuration page and then click on `Next: permissions`.
+
+![Another AWS Account](./role-creation.png)
+
+Once you are on the next page, ignore attached policies and click on `Next: tags` (Roadie does not need to read your AWS resources, only access to your k8s cluster).
+
+Optional: Add a tag, Key: `3rdPartyIntegration` Value: `Roadie`
+
+Click `Next Review`
+
+For the `Role Name` enter: `YOUR-COMPANY-NAME-backstage-backend-role-kubernetes`
```

Comment:
We shouldn't impose a naming convention for the role since it's not necessary. A lot of companies have their own conventions to these so we can let them use those instead if they want. I'd change this as a "suggested name" or something similar

Reply:
Ok cool!